README.md

```diff
@@ -9,3 +9,8 @@ library, to write homebrew graphical programs for the Nintendo 3DS.
 
 * `citro3d-sys` - C bindings to `libcitro3d`
 * `citro3d` - safe Rust wrappers for `citro3d-sys` (WIP)
+
+## License
+
+* `citro3d-sys` is licensed under Zlib
+* `citro3d` is dual-licensed under MIT or Apache-2.0
```

Comment:
I was originally going to suggest that we stick with Zlib for `citro3d` as well, since it's derived from Zlib, but based on other discussion it doesn't seem like we need to (particularly since all of the `citro3d` code is new and didn't have any license previously).